Quaver.Shared/Screens/Gameplay/CustomAudioSampleCache.cs

```diff
@@ -0,0 +1,104 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * Copyright (c) Swan & The Quaver Team <support@quavergame.com>.
+*/
+
+using System.Collections.Generic;
+using Quaver.Shared.Database.Maps;
+using Wobble.Audio.Samples;
+
+namespace Quaver.Shared.Screens.Gameplay
+{
+    public class CustomAudioSampleCache
```

Comment:
Should be a static class.

Reply:
Where? Same folder it's currently in is fine?

Reply:
Quaver.Audio would be more fitting